CONTRIBUTING.md

```diff
@@ -85,6 +89,75 @@ Because we will only merge a Pull Request for which all tests pass. The followin
   See [About TypeScript demos](#about-typescript-demos).
 - The Pull Request title follows the pattern `[Component] Imperative commit message`. (See: [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/#imperative) for a great explanation)
 
+#### Checks and how to fix them
+
+If any of the checks fails click on the _Details_
+link and checkout the logs of the build to find out why it failed. The following
+section gives an overview what each check is responsible for.
+
+##### ci/codesandbox
+
+This task should not fail in isolation. It creates codesandboxes that use the version
```

Comment:
```suggestion
This task should not fail in isolation. It creates a CodeSandbox sandbox that uses the version
```

(A bit of a mouthful, but more technically correct?)

Reply:
```suggestion
This task should not fail in isolation. It creates multiple sandboxes on CodeSandbox.com that use the version
```
codesandbox.com also uses just the term "sandbox" e.g. `Create Sandbox` or `Create React Sandbox`.